src/components/navbar.tsx

```diff
@@ -3,63 +3,73 @@ import { Link } from 'react-router-dom';
-import { Button } from '@material-ui/core';
+// import Alert from './alertComponent/alert';
+// import { Button } from '@material-ui/core';
 
 const Navbar: React.FC = () => {
   const dispatch = useDispatch();
   const { user } = useSelector((state: StoreState) => state);
   return (
     <div className="navbar-fixed">
-      <nav className="green accent-4">
+      <nav>
         <div className="nav-wrapper">
-          <Link to="/" className="navLogo">
-            skräp
-          </Link>
-          <ul id="nav-mobile" className="right hide-on-small-only">
-            {!!user.clearance && (
-              <li className="user-profile">
-                <Link to="/account">
-                  <img
-                    src={user.image}
-                    width="20"
-                    height="20"
-                    className="border-circle"
-                    alt={`${user.firstName} ${user.lastName}`}
-                  />
+          <div>
+            <Link to="/" className="navLogo">
+              skräp
+            </Link>
+          </div>
+          <div className="navList">
+            <ul id="nav-mobile" className="right hide-on-small-only">
+              {/* <li>
+                <Button>
+                  <Alert />
+                </Button>
+              </li> */}
+              <li>
+                <Link to="/jobs">Jobs</Link>
+              </li>
+              <li>
+                <Link to="/map">Map</Link>
+              </li>
+              <li>
+                <Link to="/inbox" className={user.clearance ? '' : 'ghost'}>
+                  Inbox
+                </Link>
+              </li>
+              <li>
+                <Link to="/account" className={user.clearance ? 'ghost' : ''}>
+                  Account
                 </Link>
-                <div className="user-profile-subnav green accent-4">
-                  <button
-                    className="btn btn-small"
-                    onClick={() => dispatch(logoutThunk())}
-                    type="button"
-                  >
-                    Logout
-                  </button>
-                  <Link to={`/stripe/${user.id}`}>Stripe</Link>
-                </div>
               </li>
-            )}
-            <Button color="primary">
-              <Alert />
-            </Button>
-            <li>
-              <Link to="/jobs">Jobs</Link>
-            </li>
-            <li>
-              <Link to="/map">Map</Link>
-            </li>
-            <li>
-              <Link to="/inbox" className={user.clearance ? '' : 'ghost'}>
-                Inbox
-              </Link>
-            </li>
-            <li>
-              <Link to="/account" className={user.clearance ? 'ghost' : ''}>
-                Account
-              </Link>
-            </li>
-          </ul>
+              {!!user.clearance && (
+                <li className="user-profile">
+                  <Link to="/account">
+                    <img
```

Comment:
can you use Avatar meterial UI component?
Please check https://github.com/SkrapLitter/capstone/pull/85/files#diff-d8e493dd70874bbfb8ad08ce303a16d0R91-R95